components/Slider/examples/SliderAutolayoutExampleViewController.m

```diff
@@ -33,26 +34,27 @@ - (id)init {
   self = [super init];
   if (self) {
     self.colorScheme = [[MDCSemanticColorScheme alloc] init];
+    self.typographyScheme = [[MDCTypographyScheme alloc] init];
   }
   return self;
 }
 
 - (void)viewDidLoad {
   [super viewDidLoad];
-  // Do any additional setup after loading the view, typically from a nib.
 
   // Ensure a consistent starting state
   _materialSlider.enabled = YES;
   _vanillaSlider.enabled = _materialSlider.enabled;
   _enabledSwitch.on = _materialSlider.enabled;
   _materialSlider.value = _vanillaSlider.value;
-  
-  self.materialSlider.statefulAPIEnabled = YES;
```

Comment:
Why turn off the new API?

Reply:
If statefulAPI is on then the slider is not shown.

Reply:
That seems .... terrifyingly bad. Please file a bug with steps to reproduce.

Reply:
https://github.com/material-components/material-components-ios/issues/4918